monitor-translations/dns/rename-type.json

```diff
@@ -0,0 +1,11 @@
+{
+  "agentType": "TELEGRAF",
+  "monitorType": "dns",
+  "name": "rename-type",
+  "description": "The telegraf input plugin is named dns_query",
+  "translatorSpec": {
+    "type": "replaceStringFieldValue",
```

Comment:
I'm now wondering if we should have a convenience translator type called something like "renameType".

Reply:
I'm indifferent.  It'd be the same as this but with a hardcoded `type` field.

Reply:
Agreed, it would be an almost duplicate implementation, but the readability would help a tiny bit.

Reply:
Okay, I'll add it.  Easy enough.